src/utils/k8sUtils.py

```diff
@@ -194,7 +194,7 @@ def GetLog(jobId):
                 if "status" in item and "containerStatuses" in item["status"] and "containerID" in item["status"]["containerStatuses"][0]:
                     containerID = item["status"]["containerStatuses"][0]["containerID"].replace("docker://", "")
                     log["containerID"] = containerID
-                    log["containerLog"] = kubectl_exec(" logs " + log["podName"])
+                    log["containerLog"] = kubectl_exec(" logs %s --tail=%s" % (log["podName"], str(tail)))
```

Comment:
Maybe we should allow to get all logs if tail = None
if tail is not None:
    log["containerLog"] = kubectl_exec(" logs %s --tail=%s" % (log["podName"], str(tail)))
else:
    log["containerLog"] = kubectl_exec(" logs " + log["podName"])

Reply:
ok, fair enough.